File: Report_1/Step_2/nonUseFunc_Or_useFunc/nonUseFunc_Or_useFunc.c
#include <stdio.h>

int useFunc(int a)
{
	int i, sum = 0;

	for (i = 1; i <= a; i++)
		sum += i;

	return sum;
}

int main()
{
	int result1 = 0, result2 = 0;
	int i;

	result1 = useFunc(10);

	for (i = 1; i <= 10; i++)
		result2 += i;

	printf("UseFunc    : %d\n", result1);
	printf("nonUseFunc : %d\n", result2);
	// UseFunc    : 55
	// nonUseFunc: 55

	return 0;
}

File: Report_1/Step_2/nonUseFunc_Or_useFunc/nonUseFunc_Or_useFunc.java


public class NonUseFunc_Or_UseFunc {

	static int useFunc(int n) {
		int sum = 0;

		for (int i = 1; i <= n; i++)
			sum += i;

		return sum;
	}

	public static void main(String[] args) {
		int nonUseFunc = 0;

		for (int i = 1; i <= 10; i++)
			nonUseFunc += i;

		System.out.println("useFunc : " + useFunc(10));
		// useFunc : 55

		System.out.println("nonUseFunc : " + nonUseFunc);
		// nonUseFunc : 55

	}

}
